test/integration/reporters.js

```diff
@@ -60,4 +60,45 @@ describe('reporters', function() {
       });
     });
   });
+
+  describe('json', function() {
+    it('prints test cases with --reporter-options output', function(done) {
+      var randomStr = crypto.randomBytes(8).toString('hex');
+      var tmpDir = os.tmpDir().replace(new RegExp(path.sep + '$'), '');
+      var tmpFile = tmpDir + path.sep + 'test-json-reporter-options-' + randomStr + '.json';
+
+      var args = ['--reporter=json', '--reporter-options', 'output=' + tmpFile];
+      run('passing.js', args, function(err, result) {
+        if (err) return done(err);
+
+        var json = fs.readFileSync(tmpFile, 'utf8');
+        fs.unlinkSync(tmpFile);
+        var results = JSON.parse(json);
+        assert(results.passes && results.passes.length === 2, 'JSON did not contain the expected results.');
```

Comment:
`results.passes && results.passes.length === 2` -> `results.passes.length === 2`.

This way we'll know if it failed for a deeper reason (if it didn't even have a `passes` array).